### PR TITLE
Fast sync sign hodloevsky

### PR DESCRIPTION
**Pull request by SBrooks24**
Signed and verified utxo-snapshot-bitcoin-mainnet-744358

### CODE REVIEW

File: contrib/FastSync/sigs/sbrooks24.utxo-sets.asc [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+fab994299273080bf7124c8c45c4ada867974ca747900178496a69e450cf713f  utxo-snapshot-bitcoin-mainnet-551636.tar
+eabaaa717bb8eeaf603e383dd8642d9d34df8e767fccbd208b0c936b79c82742  utxo-snapshot-bitcoin-testnet-1445586.tar
+8e18176138be351707aee95f349dd1debc714cc2cc4f0c76d6a7380988bf0d22  utxo-snapshot-bitcoin-mainnet-565305.tar
+58f65fc3e2abc367602c07ab0e7ae4a042e6acf2bc026623c7322d1886f52098  utxo-snapshot-bitcoin-mainnet-585333.tar
+52f0fc62dd28d016f49a75c22a6fa0827efc730f882bfa8cbc5ef96736d12100  utxo-snapshot-bitcoin-mainnet-609375.tar
+0afe8ca07e0ad2b538f43fc3ada89500b3e8594e8c16e0813c30d25fcff0d5a3  utxo-snapshot-bitcoin-mainnet-651474.tar
+587ac27b1929123313f842edbdb798afe116548e2a1bddea46d8d7068a6bc156  utxo-snapshot-bitcoin-mainnet-680891.tar
+79d5b28fb140c23602bebaf90e829fc89aef2bb95cb2a5d492cd818f22f01836  utxo-snapshot-bitcoin-mainnet-699714.tar
+58b3c9c8a3e5f7b3449521e8d7f8435db2e948918063cb7f5b0e635862e03891  utxo-snapshot-bitcoin-mainnet-720179.tar
```
[the remark this answers is not in the document]
Comment (SBrooks24) [reply contrib/FastSync/sigs/sbrooks24.utxo-sets.asc]: Should be removed now, thank you.

**Review by britttttk — APPROVED** [review]

**Comment by SBrooks24** [comment]
Just a note - the .asc file was edited manually to remove the extra lines.  
If this is incorrect, and the .utxo-set file should have been edited first, and re-signed to produce an updated .asc file, then that process was followed to create this alternative branch fastsync-1: https://github.com/SBrooks24/btcpayserver-docker/pull/1

**Comment by NicolasDorier** [comment]
I think PGP doesn't care about whiteline

**Comment by NicolasDorier** [comment]
Your signature isn't good [SBrooks24](https://github.com/SBrooks24)
```
keybase pgp verify -i sigs/SBrooks24.utxo-sets.asc    
- ERROR Check sig error: openpgp: invalid signature: RSA verification failure
```

**Pull request merged**